content/docs/command-reference/run.md

```diff
@@ -227,14 +227,13 @@ $ dvc run -n my_stage './my_script.sh $MYENVVAR'
   It's used by `dvc repro` to change the working directory before executing the
   `command`.
 
-- `--no-exec` - create a stage file, but do not execute the `command` defined in
-  it, nor cache dependencies or outputs (like with `--no-commit`, explained
-  below). DVC will also add your outputs to `.gitignore`, same as it would do
-  without `--no-exec`. Use `dvc commit` to force committing existing output file
-  versions to cache.
-
-  This is useful if, for example, you need to build a pipeline quickly first,
-  and run it all at once later.
+- `--no-exec` - create a stage, but do not execute the `command` defined in it.
+  DVC will still add the outputs to `.gitignore`. If the outputs already exist
+  in the workspace, you can use `dvc commit` to force DVC to track them (as if
+  the stage had run).
+
+  This option is useful, for example, to define a full `dvc.yaml` pipeline
```

Comment:
It all still feels strange. Sequence is not correct here to my mind. 

`If the outputs already exist in the workspace, you can use `dvc commit` to force DVC to track them (as if the stage had run).` - seems to be an essential part of the explanation of how this command operates. But in fact, it's just a similar use case as the that `This option is useful` describes

Reply:
OK I changed "If the outputs already exist..." for "You can use `dvc commit` to finish the operation." (like in the other commands and options that relate to commit per our other recent PR on commit-related texts).

Reply:
but then you are loosing the point about existing outputs ...

(aslo, is it about other commands we put `This option is useful` into?)

Reply:
The point about existing outputs is quite an edge case no? Not sure we need to go over it in docs. We similarly removed it from the `import` refs.

In the case of run/repro is not AS edge as with imports, but we are already mentioning it in https://dvc.org/doc/command-reference/commit#example-executing-stage-commands-without-dvc and in https://dvc.org/doc/user-guide/how-to/add-deps-or-outs-to-a-stage ("If the `data/raw.csv` or `data/validate` files already exist, we can use `dvc commit`". It's more about commit than about run/repro I think.

But if you prefer I can mention it here and in repro again.

Reply:
> is it about other commands we put This option is useful into?

Oops, didn't understand the question, sorry. We do have a "This is useful, for example..." paragraph here still though, if that's what you were referring to.